### PR TITLE
Add "Ad-Hoc Mesh Networks with Satellites" project idea

### PR DESCRIPTION
**Pull request by mwarning**
The task for the student would be to create a test tsetup to test mesh routing protocols (batman-adv, babel, ..) with satellites.

### CODE REVIEW

File: collections/_projects/freifunk-on-satellites.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+desc: Create a virtual satellite scenario to test various mesh routing protocols. 
+maskdown: freifunk-for-satellites.md
+collaborating_projects:
+  - freifunk
+developers_involved: []
+difficulty: medium
+status: open
+email: moritzwarning@web.de
+initiatives:
+  - GSoC
```
Comment (andibraeu) [inline collections/_projects/freifunk-on-satellites.md:12]: please add `GSoC2022`, too

File: collections/_projects/freifunk-on-satellites.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
```
Comment (andibraeu) [inline collections/_projects/freifunk-on-satellites.md:1]: we need the metadata `size` here.. please set
`size: half-time` or `size: full-time`

**Review by andibraeu — CHANGES_REQUESTED** [review]

**Review by andibraeu — CHANGES_REQUESTED** [review]
please remove the testbed project file, if it is not needed anymore

**Comment by mwarning** [comment]
@andibraeu I have added a MR to remove the testbed project: https://github.com/freifunk/projects.freifunk.net-contents/pull/71

**Comment by AmeyaKaloge** [comment]
hey there, 
I am Amey. I have read about this project and I really want to contribute to this particular project also I want to participate this years GSoC. Kindly help me to start my contribution.

**Comment by mwarning** [comment]
hi @AmeyaKaloge, nice to see you reaching out already.
If Freifunk will be selected (we will know on March 7) as participation GSoc organisation, then people can officially apply for projects starting April 4 (see https://developers.google.com/open-source/gsoc/timeline).

Feell free to email me and we can talk about what you are interested in and what you would like to achieve.

**Pull request merged**